app/src/main/java/com/example/recipeapp/fragments/RecipeSearchFragment.java

```diff
@@ -119,4 +138,20 @@ public void onFailure(int statusCode, Headers headers, String response, Throwabl
             }
         });
     }
+
+    private void getUser() {
+        ParseQuery<ParseUser> query = ParseQuery.getQuery(ParseUser.class);
+        query.whereEqualTo(User.KEY_OBJECT_ID, ParseUser.getCurrentUser().getObjectId());
+        query.include(User.KEY_INGREDIENT_ARRAY);
+        query.include(User.KEY_PROFILE_IMAGE);
+
+        Log.i(TAG, "User id: " + ParseUser.getCurrentUser().getObjectId());
+        try {
+            ParseUser parseUser = query.get(ParseUser.getCurrentUser().getObjectId());
+            currentUser = new User(parseUser);
+            return;
```

Comment:
no need to call return. The method will return here automatically since it is the last executable line of code.